src/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/useCreateSettingsBackupSecondFactorsMutation.ts

```diff
@@ -10,7 +10,7 @@ export const useCreateSettingsBackupSecondFactors = () => {
   const { relayEnvironment } = useSystemContext()
 
   const submitCreateSettingsBackupSecondFactors = (
-    input: CreateBackupSecondFactorsInput = {}
+    input: CreateBackupSecondFactorsInput
```

Comment:
if this input is required, any reason why we don't have `CreateBackupSecondFactorsInput!` (note the `!`)

Reply:
(Non-blocking, just thinking out loud that that's the more correct type-safe way of doing this0

Reply:
@damassi I tried that by the editor keeps auto-correcting it to `!CreateBackupSecondFactorsInput`. Here is what I see:

<img width="614" alt="Screen Shot 2022-07-25 at 2 39 35 PM" src="https://user-images.githubusercontent.com/29984068/180850888-2ff11625-4471-4e84-81bd-1498b7a2114b.png">

Reply:
@damassi This is vanilla TS, I think you're thinking about GraphQL types + nullability? This should already be sufficient as-is since its not an optional argument.

Reply:
Ah, apologies, to elaborate, this should be related to the internals of the generated type as it relates to the query itself. The `!` would apply to the graphql query, not the typescript definition.

Reply:
yup exactly @mzikherman - referring to graphql

Reply:
@damassi do you mean this? https://github.com/artsy/force/blob/9fb0f70c7abda2db06a79d80d4a77dea68c53334/src/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/useCreateSettingsBackupSecondFactorsMutation.ts#L30

Reply:
yep 👍